packages/core-ethereum/crates/core-ethereum-rpc/src/client.rs

```diff
@@ -55,8 +57,7 @@ impl<Req: HttpPostRequestor + Debug + Clone> Clone for JsonRpcProviderClient<Req
     }
 }
 
-#[cfg_attr(target_arch = "wasm32", async_trait(?Send))]
```

Comment:
I was leaving them in in some places, but dunno if trait limitations to `Send` (albeit non-sensical) should stay.

Reply:
e.g. ethers-rs has this over each async-trait, so building on a WASM target and using ethers-rs means we have to play their game too. 
Since wasm runtimes are unlikely multithreaded (yet), the non-Send on futures kinda makes sense for wasm targets.

Reply:
Yes, true. But `Send` is a stricter limitation than without it.